### PR TITLE
Northern Ireland update v1.5

### PR DESCRIPTION
**Pull request by marliekeverweij**
Updated:
- energy balance final demands HHV -> LHV
- oil demands agriculture and industry increased
- number of houses
- insulation built environment
- biomass shares built environment

### CODE REVIEW

File: datasets/UKNI01_northern_ireland/graph_values.yml [hunk 1]
```diff
@@ -1,35 +1,35 @@
 ---
 agriculture_final_demand_wood_pellets:
-  demand: 997.0
+  demand: 918.68
 agriculture_final_demand_steam_hot_water:
   demand: 0.0
 agriculture_final_demand_crude_oil:
-  demand: 88.13305584
+  demand: 6456.3
```
Comment (michieldenhaan) [inline datasets/UKNI01_northern_ireland/graph_values.yml:7]: Double-check, is this correct?

File: datasets/UKNI01_northern_ireland/graph_values.yml [hunk 2]
```diff
@@ -301,13 +301,13 @@ industry_final_demand_for_other_crude_oil_non_energetic:
 industry_useful_demand_for_chemical_refineries_crude_oil_non_energetic:
   demand: 0.0
 industry_final_demand_network_gas:
-  demand: 4709.837812
+  demand: 4238.85
 industry_chp_ultra_supercritical_coal:
   demand: 0.0
 industry_final_demand_crude_oil:
-  demand: 17026.79406
+  demand: 19592.73
```
Comment (michieldenhaan) [inline datasets/UKNI01_northern_ireland/graph_values.yml:308]: Correct? Most other demands go down

Comment (marliekeverweij) [reply datasets/UKNI01_northern_ireland/graph_values.yml:308]: Yes these are correct, thanks for checking. I will double-check with them tomorrow, but this is the data they have given me :)

File: datasets/UKNI01_northern_ireland/graph_values.yml (hunk 1 above)
Comment (marliekeverweij) [reply datasets/UKNI01_northern_ireland/graph_values.yml:7]: Yes these are correct, thanks for checking. I will double-check with them tomorrow, but this is the data they have given me :)

**Review by michieldenhaan — APPROVED** [review]

**Pull request merged**